### DISCUSSION TITLE
[Fluentui v9] boxShadow cannot override the style by [data-fui-focus-visible] for button

### DISCUSSION THREAD

**Discussion by flora8984461** [microsoft/fluentui#27638] [Q&A] [answered]
I want to use my own focus style to meet our design requirement. The button component has boxShadow applied when with [data -fui-focus-visible]. Adding boxShadow to makeStyles without id won't work, I have to indicate the id for the button to override the boxShadow. As the screenshot shows: 

// not working:
<img width="938" alt="image" src="https://user-images.githubusercontent.com/39588094/232926525-d1824467-ba2e-461d-bbc0-6d8ac9e90701.png">

// working after I use id:
<img width="833" alt="image" src="https://user-images.githubusercontent.com/39588094/232927200-efb907c3-0d0b-4485-8cfe-8e64f10ea105.png">

class with attribute name seem to have a higher priority. 

And the codesandbox link is here: 
https://codesandbox.io/s/elastic-thunder-t63vyj?file=/example.tsx

Is this a known issue and reported by anyone? 🤔 And any thoughts on that to improve it? 

Thank you for sharing any inputs. 😊

**Comment by layershifter** [comment, marked as answer]
FYI Here is how you can override it, https://github.com/microsoft/fluentui/issues/27943#issuecomment-1559425696
